optimum/intel/__init__.py

```diff
@@ -87,6 +87,9 @@
     _import_structure["openvino"].extend(
         [
             "OVModelForAudioClassification",
+            "OVModelForAudioFrameClassification",
```

Comment:
Done, thanks again, I hope I have them all now! 
I also changed an import in test_modeling.py to use the import from optimum.intel so that this is tested too.

Reply:
> I also changed an import in test_modeling.py to use the import from optimum.intel so that this is tested too.

Excellent point, thanks !